flexmeasures/ui/templates/crud/asset.html

```diff
@@ -66,9 +60,19 @@
                 </div>
                 {% endif %}
             </div>
-            <form action="/assets/{{ asset.id }}/status" method="get">
-                <button id="asset-status-button" class="btn btn-sm btn-responsive" type="submit">Asset status</button>
-            </form>
+            <div class="header-action-button">
+                <div class="">
+                    <form action="/assets/{{ asset.id }}/status" method="get">
+                        <button id="asset-status-button" class="btn btn-sm btn-responsive" type="submit">Asset status</button>
+                    </form>
+                </div>
+                <div class="">
```

Comment:
Sure this should have no class?

In my browser, the two buttons have a small white space between them, while the action buttons above them do not. That does not look good.